frameworks/D/vibed/install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+sudo wget http://master.dl.sourceforge.net/project/d-apt/files/d-apt.list -O /etc/apt/sources.list.d/d-apt.list
```

Comment:
You'll have to do this without `sudo` - we can't accept further PRs that utilize sudo in their install.sh scripts (we are busy now trying to remove all instances of this)